compiler/src/iree/compiler/Codegen/LLVMGPU/test/config_matvec.mlir

```diff
@@ -370,76 +270,3 @@ func.func @not_vmt() {
 //  CHECK-SAME:     translation_info = #[[$TRANSLATION]]
 //       CHECK:   linalg.generic
 //  CHECK-SAME:       lowering_config = #iree_gpu.lowering_config<{reduction = [0, 0, 8], thread = [1, 128, 0], workgroup = [1, 128, 1]}>
-
-// -----
-
-func.func @dynamic_parallel_dims(%dynsize : index, %input : tensor<4x?x4096xf16>) -> tensor<4x?xf32> {
```

Comment:
None of these tests should be deleted, they need to go through vector distribute now